hiss/application/models.py

```diff
@@ -286,3 +288,48 @@ def clean(self):
             raise exceptions.ValidationError("First name can't contain any numbers")
         if any(char.isdigit() for char in self.last_name):
             raise exceptions.ValidationError("Last name can't contain any numbers")
+
+
+class WaveManager(models.Manager):
+    def next_wave(self, dt: datetime.datetime = timezone.now()):
+        """
+        Returns the next INACTIVE wave, if one exists. For the CURRENT active wave, use
+        `active_wave`.
+        """
+        qs = self.get_queryset().filter(start__gt=dt).order_by("start")
+        return qs.first()
+
+    def active_wave(self, dt: datetime.datetime = timezone.now()):
```

Comment:
See comment regarding usage of `django.utils.timezone` instead of `datetime`

```suggestion
    def active_wave(self, dt: timezone.datetime = timezone.now()):
```